windows/.bazelrc

```diff
@@ -1,22 +1,49 @@
 # Windows/Envoy specific Bazel build/test options.
 
+build --action_env=PATH
+build --action_env=TMPDIR
+
+# see top-level .bazelrc for explanation
+startup --host_jvm_args=-Xmx2g
+
 build --define signal_trace=disabled
 build --define hot_restart=disabled
 build --define tcmalloc=disabled
+build --define manual_stamp=manual_stamp
+
 build --workspace_status_command="bash bazel/get_workspace_status"
+build --host_javabase=@bazel_tools//tools/jdk:remote_jdk11
+build --javabase=@bazel_tools//tools/jdk:remote_jdk11
+build --enable_platform_specific_config
 
-# Required to work around quiche build defect
-# Unguarded gcc pragmas are not recognized by MSVC
-build --copt="/wd4068"
+# Experimental: We already have absl in the build, define absl=1
+# to tell googletest to use absl for backtrace (this appears to be
+# broken on msvc-cl dbg builds)
+build --define absl=1
 
 # Should not be required after upstream fix, see issue;
 # https://github.com/bazelbuild/rules_foreign_cc/issues/301
-build --copt="-DCARES_STATICLIB"
 build --copt="-DNGHTTP2_STATICLIB"
-build --copt="-DCURL_STATICLIB"
+
+# Required to work around quiche build defect
+# Unguarded gcc pragmas are not recognized by MSVC
+build:msvc-cl --copt="/wd4068"
```

Comment:
Is this the walkaround for https://github.com/envoyproxy/envoy/issues/10239?

Reply:
Well I don't think it is from the comment... How did you walk around that failure?

Reply:
Nope, nothing to do with 10239, that fix is not offered in this PR. That fix will become required when we begin compiling //test/... (which this PR doesn't attempt to do). This PR simply gets us compiling //source/exe:envoy-static

You can see the workaround to 10239 at https://github.com/greenhouse-org/envoy/commit/66eff7c1e12544a37792140039867481b9bc91f8 which we will offer in the next PR that addresses compiling //test/...

This fix above resolves gcc pragmas which cause fatal warnings-as-errors failure when we trip over the gcc pragmas in poorly constructed quiche headers. See;
https://docs.microsoft.com/en-us/cpp/error-messages/compiler-warnings/compiler-warning-level-1-c4068?view=vs-2019

Reply:
Ah, no that was a comment that snuck in from when we were trying to fix the quiche thing before, will remove the comment (that warning is around unrecognized pragmas: https://docs.microsoft.com/en-us/cpp/error-messages/compiler-warnings/compiler-warning-level-1-c4068?view=vs-2019)

Reply:
Note it is extra confusing because we transform the source tree from clang pragmas into gcc pragmas, both of which cannot be understood by msvc-cl. (clang-cl, when we get that working, would probably recognize the clang flavor of the pragmas)